deploy/mixed-firmware/x86_64_x11ssh_qemu_linuxboot.defconfig

```diff
@@ -284,7 +285,9 @@ CONFIG_TMPFS=y
 CONFIG_HUGETLBFS=y
 CONFIG_CONFIGFS_FS=y
 # CONFIG_MISC_FILESYSTEMS is not set
-# CONFIG_NETWORK_FILESYSTEMS is not set
+CONFIG_9P_FS=y
+CONFIG_9P_FS_POSIX_ACL=y
+CONFIG_9P_FS_SECURITY=y
```

Comment:
Do you remember where in the kernel's menuconfig you have enabled this settings?